orchestration-shared/src/main/java/uk/gov/di/orchestration/shared/entity/OrchSessionItem.java

```diff
@@ -24,6 +25,7 @@ public enum AccountState {
     private String verifiedMfaMethodType;
     private String rpPairwiseId;
     private AccountState isNewAccount;
+    private String internalCommonSubjectIdentifier;
```

Comment:
rename to `internalCommonSubjectId` if that's the nomenclature you want to go with

Reply:
fixed